MoreLinq/ZipImpl.cs

```diff
@@ -1,97 +0,0 @@
-#region License and Terms
-// MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2018 Leandro F. Vieira (leandromoh). All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-#endregion
-
-namespace MoreLinq
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Collections;
-
-    static partial class MoreEnumerable
-    {
-        delegate TResult Folder<in T, out TResult>(params T[] args);
-
-        static IEnumerable<TResult> ZipImpl<T1, T2, T3, T4, TResult>(
```

Comment:
Removed as code is simpler when re-implemented for equi-, -shortest, and -longest.